o4/o4.py

```diff
@@ -1080,7 +1080,7 @@ def o4_status(changelist, depot, check_all, quick):
            cur=cur,
            check_all=check_all,
            quick=quick)
-    if cur is None:
+    if not cur:
```

Comment:
I seem to recall there was a difference between None and a number here.

Reply:
I see that there is, but I don't think it was anticipated that the number would be zero.